src/test/java/com/wikia/webdriver/common/driverprovider/NewDriverProvider.java

```diff
@@ -112,12 +106,11 @@ private static EventFiringWebDriver getIEInstance() {
 
   private static EventFiringWebDriver getAndroidInstance() {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "resources" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)